### PR TITLE
Add a benchmark that measures gRPC server performance with `ghz`

### PR DESCRIPTION
**Pull request by ikhoon**
Motivation:

https://github.com/bojand/ghz is a popular benchmarking and load tool
for gRPC.  https://github.com/LesnyRumcajs/grpc_bench also uses `ghz`
as its load tool. It would be convenient to run benchmarks in Armeria
without publishing a snapshot version to local or Maven central.

Modifications:

- Add `benchmarks:grpc:ghz` module
- Move `benchmarks` to `benchmarks:jmh`
- Add a simple server that is on a par with
  https://github.com/LesnyRumcajs/grpc_bench server.
- Add `bench.sh` that loads the server using `ghz`

Result:

Easily measure gRPC server's performance using `ghz`

### CODE REVIEW

**Review by minwoox — APPROVED** [review]
LGTM once @anuraaga's comment is addressed. 👍

File: it/grpc/ghz/src/main/proto/hello.proto [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+syntax = "proto3";
+
+option java_multiple_files = true;
+package com.linecorp.armeria.grpc.ghz;
```
[the remark this answers is not in the document]
Comment (ikhoon) [reply it/grpc/ghz/src/main/proto/hello.proto]: Makes sense. Updated.

File: settings.gradle [hunk 2]
```diff
@@ -70,6 +70,7 @@ includeWithFlags ':it:builders',                    'java'
 includeWithFlags ':it:context-storage',             'java'
 includeWithFlags ':it:dgs',                         'java'
 includeWithFlags ':it:grpcweb',                     'java', 'akka-grpc_2.13'
+includeWithFlags ':it:grpc:ghz',                    'java', 'relocate'
```
Comment (trustin) [inline settings.gradle:73]: Why does this need to be relocated?

**Review by trustin — CHANGES_REQUESTED** [review]
I'm not sure if it makes sense to put benchmarks under `:it`. How about moving `:benchmarks` to `:benchmarks:jmh` and move this module to `:benchmarks:ghz`?

File: settings.gradle (hunk 2 above)
Comment (ikhoon) [reply settings.gradle]: Nope. It does not need to be tested with shaded code.

**Pull request merged**